store/deposit-dates.js

```diff
@@ -73,16 +71,24 @@ class DepositDates extends Store {
   }
 
   @action
-  async getCrossDepositLag() {
+  async retrieveCrossDepositLag() {
     const { data } = await this.request(this.crossDepositLagUrL)
     this.crossDepositLag = data
   }
 
   @action
-  async getPublicationDatesValidate() {
+  async retrievePublicationDatesValidate() {
     const { data } = await this.request(this.publicationDatesValidateUrl)
     this.publicationDatesValidate = data
   }
+
+  async retrieve() {
+    // Doing it sequentially because of the API suffering
+    // from heavy concurrent requests
+    await this.retrieveDepositTimeLag()
```

Comment:
It should be executed differently otherwise if the first request fails the following will fail too. either wrap every request with try/catch or try to find a better solution. 

To be honest I am against doing these things because it looks to me like it's against javascript. This is something which should backend handle better. :(  If it is needed I can cope with it.

Reply:
```js
 await Promise.allSettled(retrieveDepositTimeLag(), ...)
```
Like this

Reply:
That doesn't guarantee sequential order.

Reply:
Yeah, you are right.

Can be chained in
```js
this.retrieveDepositTimeLag().finally(...).finally(...)
```

if it's possible

Reply:
Seems to work as expected

```js
let i = 0;

const exec = (timeout = 500) => 
  new Promise((resolve, reject) => {
    i++;
    console.log('call', i);
    setTimeout(() => {
      console.log('reject', i);
      reject(new Error(`Reject ${i}`));
    }, timeout);
  });

exec().finally(exec).finally(exec).catch(() => {});
```